### PR TITLE
:cow2: GP Finder templates

### PR DESCRIPTION
**Pull request by beseven**
Happy for this to be reviewed after your common CI changes have gone in and @st3v3nhunt removing of the dev environment

### CODE REVIEW

**Comment by beseven** [comment]
@st3v3nhunt I'll add the `gp-finder` to the lb in this request too

File: templates/c2s-gp-finder/config.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: C2S - GP Finder
+description: |
+  Connecting to Services - GP Finder (require separate loadbalancer stack or traefik lb running in the stack)
+version: v1
+category: nhsuk beta
+projectURL: https://github.com/nhsuk/nhsuk-rancher-templates
```
Comment (st3v3nhunt) [inline templates/c2s-gp-finder/config.yml]: This should be https://github.com/nhsuk/gp-finder

**Review by st3v3nhunt — APPROVED** [review]

**Comment by st3v3nhunt** [comment]
@neilbmclaughlin I'll leave the final review and merge to you.

@beseven I'll update the other catalogs to have the GitHub repo url for `projectURL` in `config.yml`. Not sure if you want to add the `gp-finder` stuff to the LB as part of this PR or do another to merge into this branch?

File: templates/c2s-gp-finder/config.yml (hunk 1 above)
Comment (beseven) [reply templates/c2s-gp-finder/config.yml]: Hm, that might be the reason the lb/c2s0 template was out of date as the config was pointing at the same url?

**Review by neilbmclaughlin — APPROVED** [review]

**Pull request merged**